cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -735,3 +756,20 @@ impl<I: VCodeInst> PrettyPrint for VCode<I> {
         s
     }
 }
+
+/// Indicates that a constant value has been used and will be emitted later.
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
+pub struct UsedConstant(u32);
+entity_impl!(UsedConstant);
```

Comment:
I'm trying to think of a better name for this -- would just `VCodeConstant` be more fitting, do you think, since they live in the `VCode`?